pkg/chart/chart_test.go

```diff
@@ -583,10 +587,10 @@ func getFakeClientRequests(t *testing.T, c HTTPClient) []*http.Request {
 
 func TestGetChart(t *testing.T) {
 	target := Details{
-		RepoURL:     "http://foo.com/",
-		ChartName:   "test",
-		ReleaseName: "foo",
-		Version:     "1.0.0",
+		AppRepositoryResourceName: "foo-repo",
```

Comment:
[Not important] I see we're reusing `foo-repo` on the `chUtils Chart` initialization. Shall we move it to a constant?

Reply:
Yep, done... thanks.